### DISCUSSION TITLE
Operation via MQTT does not want to work

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#22277] [Support request]
Hallo

ich habe einen Mqtt Broker angelegt der auch funktioniert. Mit dem Tasmota Device Manger zumindestens.

Jetzt wollt ich das mit dem Smartphone ausprobieren.

Dazu habe ich IOT MQTT Panel auf dem Smartphone installiert.
Eine Verbidung zu Broker ist hergestellt.Dann habe ich ein Panel angelegt um den Shelly zu bedienen dem ich das Tobic Hobby gegeben habe. Im Panel tat ich das gleiche. Als Paylod on habe ich "cmnd/Hobby/ POWER on" 

Als Payload off habe ich "cmd/Hobby/POWER off" eingetragen.Aber der Shelly schaltet nicht.

Was für einen Fehler habe ich gemacht oder habe ich was vergessen?
![Hobby Shelly](https://github.com/user-attachments/assets/b9f16ba0-601a-4498-a032-2c8042b2d38b)
![Screenshot_IoT MQTT Panel](https://github.com/user-attachments/assets/1a0aa105-24eb-45f2-967f-961607b44452)

**Comment by sfromis** [comment]
Topic needs to be `cmnd/....` and payload on/off.

**Comment by dippes** [comment]
Is that what you mean?

Does not work
![Screenshot_1_IoT MQTT Panel](https://github.com/user-attachments/assets/ad0b41f6-94a0-4011-8525-89a0826f09ad)

**Reply by Noschvie** [reply]
The topic have to be "cmnd/Hobby/POWER"

**Reply by dippes** [reply]
Danke funktioniert
